node/src/rpc/eth.rs

```diff
@@ -109,7 +111,7 @@ where
 {
 	use fc_rpc::{
 		Eth, EthApiServer, EthDevSigner, EthFilter, EthFilterApiServer, EthPubSub,
-		EthPubSubApiServer, EthSigner, Net, NetApiServer, TxPoolApiServer, Web3, Web3ApiServer,
+		EthPubSubApiServer, EthSigner, Net, NetApiServer, Web3, Web3ApiServer,
 	};
```

Comment:
Looks like [these lines](https://github.com/paritytech/frontier/pull/1104/files#diff-10dbae62d0fd1bd075ca0e84312de153926ca883ef247536affd3b6e1a9d9429R114-R115) are missing, no ?

Reply:
Fixed in [2d1f12d](https://github.com/paritytech/frontier-parachain-template/pull/15/commits/2d1f12d0f077411b14c4d48b1c31880d4a0bea0f).

It's weird... when building without the `txpool` feature I saw no compilation warning about unused imports.